deploy/11_approve_first_risk_manager.js

```diff
@@ -0,0 +1,17 @@
+module.exports = async function ({ getNamedAccounts, deployments }) {
+  const { execute, log } = deployments
+  const { deployer } = await getNamedAccounts()
+  const RiskManagerV1 = await deployments.get("RiskManagerV1")
+
+  log(`approving first risk manager ${RiskManagerV1.address}`)
+
+  await execute(
+    "CoveragePool",
+    { from: deployer },
+    "approveFirstRiskManager",
+    RiskManagerV1.address
+  )
```

Comment:
We need to check if the risk manager is already approved so we won't get a revert on deployment retry.

```suggestion
  const isRiskManagerApproved = await read(
    "CoveragePool",
    { from: deployer },
    "approvedRiskManagers",
    RiskManagerV1.address
  )

  if (!isRiskManagerApproved) {
    log(`approving first risk manager ${RiskManagerV1.address}`)

    await execute(
      "CoveragePool",
      { from: deployer },
      "approveFirstRiskManager",
      RiskManagerV1.address
    )
  } else {
    log(`risk manager ${RiskManagerV1.address} is already approved`)
  }
```

Reply:
503a296c6685a4796638fc7a988779dc6cd4ef28